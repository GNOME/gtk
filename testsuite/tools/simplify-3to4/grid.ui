<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow" id="window1">
    <child>
      <object class="GtkGrid" id="grid1">
        <property name="visible">True</property>
        <child>
          <object class="GtkToggleButton" id="button1">
            <property name="label" translatable="yes">Hello World!</property>
            <property name="visible">True</property>
            <property name="active">True</property>
            <property name="receives_default">True</property>
            <property name="expand">True</property>
            <property name="margin">10</property>
          </object>
          <packing>
            <property name="left_attach">0</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkCheckButton" id="button2">
            <property name="label" translatable="yes">Hello World!</property>
            <property name="visible">True</property>
            <property name="active">False</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">1</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkRadioButton" id="button3">
            <property name="label" translatable="yes">Hello World!</property>
            <property name="visible">True</property>
            <property name="active">False</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">2</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkRadioButton" id="button4">
            <property name="label" translatable="yes">Hello World!</property>
            <property name="visible">True</property>
            <property name="active">True</property>
            <property name="group">button3</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">3</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkRadioButton" id="button4a">
            <property name="label" translatable="yes">Hello World!</property>
            <property name="visible">True</property>
            <property name="group">button4</property>
            <property name="draw-indicator">False</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">3</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkSwitch" id="button5">
            <property name="visible">True</property>
            <property name="active">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">4</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkSwitch" id="button6">
            <property name="visible">True</property>
            <property name="active">False</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
          </object>
          <packing>
            <property name="left_attach">5</property>
            <property name="top_attach">0</property>
          </packing>
        </child>

        <child>
          <object class="GtkButton" id="button7">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="label">Text Button</property>
          </object>
          <packing>
            <property name="left_attach">0</property>
            <property name="top_attach">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkButton" id="button8">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
          </object>
          <packing>
            <property name="left_attach">1</property>
            <property name="top_attach">1</property>
            <property name="width">1</property>
            <property name="height">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkButton" id="button9">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
          </object>
          <packing>
            <property name="left_attach">2</property>
            <property name="top_attach">1</property>
            <property name="width">2</property>
            <property name="height">2</property>
          </packing>
        </child>
      </object>
    </child>
  </object>
</interface>
