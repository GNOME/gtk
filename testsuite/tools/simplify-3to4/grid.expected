<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkWindow" id="window1">
    <property name="child">
      <object class="GtkGrid" id="grid1">
        <child>
          <object class="GtkToggleButton" id="button1">
            <property name="label" translatable="1">Hello World!</property>
            <property name="active">1</property>
            <property name="receives_default">1</property>
            <property name="hexpand">1</property>
            <property name="vexpand">1</property>
            <property name="margin-start">10</property>
            <property name="margin-end">10</property>
            <property name="margin-top">10</property>
            <property name="margin-bottom">10</property>
            <layout>
              <property name="column">0</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkCheckButton" id="button2">
            <property name="label" translatable="1">Hello World!</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">1</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkCheckButton" id="button3">
            <property name="label" translatable="1">Hello World!</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">2</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkCheckButton" id="button4">
            <property name="label" translatable="1">Hello World!</property>
            <property name="active">1</property>
            <property name="group">button3</property>
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">3</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkToggleButton" id="button4a">
            <property name="label" translatable="1">Hello World!</property>
            <property name="group">button4</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">3</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkSwitch" id="button5">
            <property name="active">1</property>
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">4</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkSwitch" id="button6">
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <layout>
              <property name="column">5</property>
              <property name="row">0</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkButton" id="button7">
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <property name="label">Text Button</property>
            <layout>
              <property name="column">0</property>
              <property name="row">1</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkButton" id="button8">
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
            <layout>
              <property name="column">1</property>
              <property name="row">1</property>
            </layout>
          </object>
        </child>
        <child>
          <object class="GtkButton" id="button9">
            <property name="focusable">1</property>
            <property name="receives_default">1</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
            <layout>
              <property name="column">2</property>
              <property name="row">1</property>
              <property name="column-span">2</property>
              <property name="row-span">2</property>
            </layout>
          </object>
        </child>
      </object>
    </property>
  </object>
</interface>
